ramp-database/ramp_database/tools/tests/test_submission.py

```diff
@@ -65,6 +65,7 @@
 from ramp_database.tools.submission import submit_starting_kits
 
 HERE = os.path.dirname(__file__)
+ID_SUBMISSION = 7
```

Comment:
Is it supposed to be the same as the one in `ramp-database/ramp_database/model/tests/test_submission.py`?
If yes, maybe define it uniquely in `ramp-database/ramp_database/testing.py`

Reply:
could be... but I think it's easier to debug if you have the variable in the same testing file.. 
if it's very important to you I will change but I am slightly leaning against it.

Reply:
It is not that important. It is just that when you have constant, I find it better to not have multiple definitions. But if it is not critical (as it is in the tests), do what you think is the best :)